benchmark/stream_buffer.cpp

```diff
@@ -63,6 +63,30 @@ StreamBuffer_Front(benchmark::State& state)
     }
 }
 
+static void
+StreamBuffer_Pop(benchmark::State& state)
+{
+    quicr::StreamBuffer<std::uint8_t> buffer;
+    std::vector<uint8_t> bytes(1'000'000, 0);
+    buffer.Push(bytes);
+
+    for ([[maybe_unused]] const auto& _ : state) {
+        buffer.Pop();
+    }
+}
+
+static void
+StreamBuffer_PopBytes(benchmark::State& state)
+{
+    quicr::StreamBuffer<std::uint8_t> buffer;
+    std::vector<uint8_t> bytes(1'000'000, 0);
+    buffer.Push(bytes);
+
+    for ([[maybe_unused]] const auto& _ : state) {
+        buffer.Pop(10);
```

Comment:
Remember this is going to pop 10 items, so for a million iterations you'll need 10 million bytes so that all calls will pop something.